src/simplay-jupyter/src/index.ts

```diff
@@ -1,5 +1,100 @@
-// Copyright (c) team-simplay
-// Distributed under the terms of the Modified BSD License.
+import { JSONObject } from '@lumino/coreutils';
+import { Widget } from '@lumino/widgets';
+import { IRenderMime } from '@jupyterlab/rendermime-interfaces';
+import { SimulationSpooler } from 'simplay-web';
 
-export * from './version';
-export * from './widget';
+import '../style/index.css';
+import { SimulationDataSerialized } from 'simplay-web/dist/SimulationDataSerialized';
+
+const SIMPLAY_CSS_COMMON_CLASS = 'jp-render-simplay';
+
+/**
+ * The MIME type for simplay.
+ *
+ */
+export const SIMPLAY_MIME_TYPE = 'application/simplay+json';
+
+/**
+ * A widget for rendering SimPlay, for usage with rendermime.
+ */
+export class RenderSimplay extends Widget implements IRenderMime.IRenderer {
+  constructor(options: IRenderMime.IRendererOptions) {
+    super();
+    console.info('JupyterLab extension simplay_jupyter is activated!');
+    this._mimeType = options.mimeType;
+    this.addClass(SIMPLAY_CSS_COMMON_CLASS);
+  }
+
+  /**
+   * Render SimPlay into this widget's node.
+   */
+  renderModel(model: IRenderMime.IMimeModel): Promise<void> {
+    const data = model.data[this._mimeType] as JSONObject;
+    console.log('data' + JSON.stringify(data));
```

Comment:
We should think about removing the console.log

Reply:
will remove it